<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="6" total="7" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-08T16:39:27 CET" name="Default suite" finished-at="2023-01-08T16:39:48 CET" duration-ms="20724">
    <groups>
    </groups>
    <test started-at="2023-01-08T16:39:27 CET" name="Default test" finished-at="2023-01-08T16:39:48 CET" duration-ms="20724">
      <class name="testSuite.PdpPomTestScripts">
        <test-method is-config="true" signature="start()[pri:0, instance:testSuite.PdpPomTestScripts@1139b2f3]" started-at="2023-01-08T16:39:27 CET" name="start" finished-at="2023-01-08T16:39:36 CET" duration-ms="9521" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- start -->
        <test-method signature="VerifyUserIsAbleToSelectVariationsOfTheProduct()[pri:0, instance:testSuite.PdpPomTestScripts@1139b2f3]" started-at="2023-01-08T16:39:36 CET" name="VerifyUserIsAbleToSelectVariationsOfTheProduct" finished-at="2023-01-08T16:39:42 CET" duration-ms="6036" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at base.CommonPOM.verifyByPageSourceContains(CommonPOM.java:109)
at testSuite.PdpPomTestScripts.VerifyUserIsAbleToSelectVariationsOfTheProduct(PdpPomTestScripts.java:171)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyUserIsAbleToSelectVariationsOfTheProduct -->
        <test-method is-config="true" signature="end(org.testng.ITestResult)[pri:0, instance:testSuite.PdpPomTestScripts@1139b2f3]" started-at="2023-01-08T16:39:43 CET" name="end" finished-at="2023-01-08T16:39:48 CET" duration-ms="5031" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=VerifyUserIsAbleToSelectVariationsOfTheProduct status=FAILURE method=PdpPomTestScripts.VerifyUserIsAbleToSelectVariationsOfTheProduct()[pri:0, instance:testSuite.PdpPomTestScripts@1139b2f3] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- end -->
      </class> <!-- testSuite.PdpPomTestScripts -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
